src/traces/scattermapbox/defaults.js

```diff
@@ -28,6 +28,15 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         return;
     }
 
+    var clusterEnabled = coerce('cluster.enabled');
```

Comment:
You should add `enabled` to the API as well. Please see my comment here: https://github.com/plotly/plotly.js/pull/5147/files#r494316347